tools/clang/test/HLSLFileCheck/hlsl/types/matrix/default-matrix-in-template.hlsl

```diff
@@ -1,5 +1,6 @@
-// RUN: %dxc -E main -T cs_6_0 %s  | FileCheck %s
+// RUN: %dxc -E main -T cs_6_0 -fcgl %s  | FileCheck %s
 
+// Check unlowered type
```

Comment:
Previously, we forgot to actually replace types in resource metadata with the lowered (or legacy translated) types.  Since this test is just checking that we generate the right type for the matrix use here, `-fcgl` check is sufficient.